src/browser/screens/Roles.styl

```diff
@@ -0,0 +1,133 @@
+.roles-screen
+  align-items center
+  background #fff url('../../../assets/img/positions/mountains.svg') no-repeat
+  background-size cover
+  background-position 0
+  display flex
+  flex-direction column
+  font-size 12px
+  overflow-y hidden
+
+  h1
+    font-size 3.5em
+    font-weight 700
+    text-transform uppercase
+    margin 10px 0 0
+
+  ul
+    display flex
+    justify-content center
+    align-items flex-end
+    height 93%
+    width 100%
+    padding 0
+    margin 0
+
+    li
+      cursor pointer
+      display flex
+      flex-direction column
+      width 20%
+      justify-content flex-end
+      position relative
+      height 100%
+
+      div
+        align-items center
+        margin-bottom 60px
+        display flex
+        flex-direction column
+        justify-content flex-end
+        height 100%
+
+        p
+          bottom 0px
+          box-sizing border-box
+          font-size 2em
+          text-align center
+          color #000
+          margin 0 0 35px 0
+          position absolute
```

Comment:
to jest absolutne wzgledem `li`, a nie `div`, o to Ci chodzilo?

Reply:
Dokładnie